core-blocks/paragraph/index.js

```diff
@@ -84,7 +84,7 @@ class ParagraphBlock extends Component {
 		this.setFontSize = this.setFontSize.bind( this );
 	}
 
-	onReplace( blocks ) {
+	onReplace( blocks, blockToSelect ) {
```

Comment:
I see this change only being applied to the paragraph block. Should it be generalised to all blocks that have an `onSplit` prop, such as headings?